src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -113,19 +113,22 @@ export default class ProcessConverter extends AbstractConverter<Process> {
   }
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private buildShapeBpmnEvent(bpmnElement: any, elementKind: ShapeBpmnElementKind, processId: string): ShapeBpmnEvent | void {
+  private buildShapeBpmnEvent(bpmnElement: any, elementKind: ShapeBpmnElementKind, processId: string): ShapeBpmnEvent {
     const eventDefinitions = this.getEventDefinitions(bpmnElement);
     const numberOfEventDefinitions = eventDefinitions.map(eventDefinition => eventDefinition.counter).reduce((counter, it) => counter + it, 0);
 
     // do we have a None Event?
-    if (numberOfEventDefinitions == 0) {
+    if (numberOfEventDefinitions === 0 && ShapeUtil.canHaveNoneEvent(elementKind)) {
```

Comment:
:question: why switching to triple =? tests are passing without the change, so I suggest to rollback the change or add a test to demonstrate the need of change